spec/features/user/user_creation_spec.rb

```diff
@@ -1,19 +1,19 @@
 require 'rails_helper'
 
 RSpec.feature "User can be created" do
-  xscenario "user can visit create account form" do
+  scenario "user can visit create account form" do
 
     visit new_user_path
 
     fill_in "Username", with: "lane"
     fill_in "Email", with: "lane@nil.com"
+    fill_in "City", with: "Denver"
     fill_in "Password", with: "password"
+
     click_button "Create Account"
 
-    expect(page).to have_content "Logged in as lane"
     expect(page).to have_link "Logout"
-    expect(page).to have_link "Cart"
-    
+
```

Comment:
@NicoleKMarino we can also add here the assertion to check that the User tables count has increased by 1 (as I commented above)